htmlmaker_postprocessing.js

```diff
@@ -95,6 +106,83 @@ fs.readFile(file, function editContent (err, contents) {
     $(this).after(match[4]);
   });
 
+  // remove Section-Blank-Page sections
+  var sectionBlankPage = $("section.blankpage");
+  sectionBlankPage.remove();
+
+  //// Strip pageBreaks preceding Section starts: Part 1 (of 2):
+  // catch & remove any page break directly preceding Section Starts
+  //  (nothing should be outside of a seciton block, but just in case)
+  var leadingPageBreak = $("section").prev(".PageBreakpb");
+  // cycle through each Page Break para directly preceding a Section Start
+  leadingPageBreak.each(function() {
+    // capture any Page Breaks directly preceding the selected one
+    var consecutiveLeadingPBs = $(this).prevUntil(":not(.PageBreakpb)").addBack();
+    consecutiveLeadingPBs.remove();
+  });
+
+  //// Strip pageBreaks preceding Section starts: Part 2 (of 2):
```

Comment:
Ah yeah, cheerio is missing support for some things. I know the snippet I put in the previous comment works generally, but it's possible cheerio is missing some of those selectors. Still, do some testing and see if you can simplify a bit.